### DISCUSSION TITLE
Scene viewer closes unexpectedly when AR is requested in a OPPO Reno5 Lite

### DISCUSSION THREAD

**Discussion by hd-esteban** [google/model-viewer#3598] [Q&A]
We've an implementation where we're using AR through Scene Viewer and we're testing in multiple devices most of them work properly but it seems to be not working properly in a OPPO Reno Lite device. I've tested and AR works with WebXR but for some reason Scene Viewer doesn't work. I've double checked and the browser has access to use the camera so it seems to be a problem between Chrome and Android to run the AR experience.

[Live Demo
](https://demo.cylindo.com/viewer/examples/misc/Blindster/2inches_Premium_Wood_Blinds.html)

- model-viewer: N/A

- [X ] Chrome, version: 102.0.5005.125

- [X ] Android

- [X] SceneViewer

![WhatsApp Image 2022-06-28 at 10 58 22 AM](https://user-images.githubusercontent.com/1881501/176233380-a1e1f929-a4da-49fd-a26e-dd5d76afd996.jpeg)
![WhatsApp Image 2022-06-28 at 10 56 48 AM](https://user-images.githubusercontent.com/1881501/176233381-344f2579-aecf-405d-b28a-3a277b75dc1f.jpeg)
![WhatsApp Image 2022-06-28 at 10 55 41 AM](https://user-images.githubusercontent.com/1881501/176233384-37786e52-ce28-405c-a37c-474694398fdc.jpeg)

**Comment by eugenecys** [comment]
Can you describe more on what you mean that AR does not work? Is there no button to go into AR mode from within Scene Viewer?

**Comment by hd-esteban** [comment]
Hi @eugenecys, thank you for your quick reply, I can see Scene viewer logo, then I can a spinning wheel loading but when the camera is open it closes unexpectedly.

![image](https://user-images.githubusercontent.com/1881501/176272776-5ceb022e-58a3-4758-b14d-ee81262877e4.png)

**Comment by base-addo** [comment]
Hello! I tested it. AR wants permission to open it with camera. When we give it to do. Camera closes unexpectedly. When we trying to open repeatedly we catched something. Here is it!!
![WhatsApp Image 2022-06-29 at 00 39 41](https://user-images.githubusercontent.com/92454096/176378830-13ce7f7f-18b2-4e46-8c1a-15a012f177ae.jpeg)
![WhatsApp Image 2022-06-29 at 00 40 36](https://user-images.githubusercontent.com/92454096/176378847-a823efdb-e99d-422a-a4bc-e3443781c3d4.jpeg)

**Comment by hd-esteban** [comment]
@eugenecys I'm facing the same issue reported by @base-addo , do you have any update on this?

**Comment by hd-esteban** [comment]
hi @elalish / @eugenecys, do you've any update on this error?

**Comment by eugenecys** [comment]
Hi, sorry that I dropped the ball on this. That's certainly unusual behavior. Does that happen with all other models surfaced by Google search? E.g. try searching for "shark" and scroll down and tap "View in 3D" and try going to AR. 

If it happens for most if not all models, specifically for the Oppo A91, it's likely due to incompatibility with ARCore. Search for Google Services for AR and reinstall it, and see if it fixes the issue.

If it still does not, can you getting the crash stack trace when the app crashes?

Thank you!

**Comment by hd-esteban** [comment]
hi @eugenecys, thank you for your feedback, it did the trick, so I think you can close this ticket.
